lib/mindwendel_web/controllers/admin/brainstorming_controller.ex

```diff
@@ -2,12 +2,17 @@ defmodule MindwendelWeb.Admin.BrainstormingController do
   use MindwendelWeb, :controller
   alias Mindwendel.Brainstormings
   alias Mindwendel.Brainstormings.Brainstorming
+  alias Mindwendel.Brainstormings.IdeaLabel
   alias Mindwendel.CSVFormatter
+  alias Mindwendel.Repo
+  import Ecto.Query
 
   plug :fetch_user
 
   def edit(conn, %{"id" => id}) do
-    brainstorming = Brainstormings.get_brainstorming_by!(%{admin_url_id: id})
+    brainstorming =
+      Brainstormings.get_brainstorming_by!(%{admin_url_id: id})
+      |> Repo.preload(labels: from(idea_label in IdeaLabel, order_by: idea_label.position_order))
```

Comment:
is the where filter on brainstorming_id already implicitly added here?

Reply:
`Repo.preload(labels: from(idea_label in IdeaLabel, order_by: idea_label.position_order))` You are talking about the `order_by` clause. Yes this is necessary to ensure the correct order coming from the database.